README.md

```diff
@@ -1,6 +1,57 @@
-# LLVM-based Solidity Compiler Toolchain
+<div align="center">
+  <img src=".github/assets/logo.png" alt="solx logo" />
+</div>
 
-This repository contains an LLVM-based Compiler Toolchain for Solidity and Yul.
+### `solx` is an experimental compiler for Solidity
+
+The goal of the project is to make state-of-the-art optimizing compilers a mainstream for Web3.
+
+We want to:
+
+- Make Ethereum more efficient.
+- Eliminate the need to write inline assembly.
+- Provide industry-grade tooling, such as interactive debugger.
+
+## Installation
+
+You can get existing pre-releases from the [Releases](https://github.com/matter-labs/solx/releases) GitHub page.
+Or, you can take a build used in [solx_demo](https://github.com/popzxc/solx_demo):
+
+- [Linux/AMD64](https://github.com/matter-labs/solx/releases/download/eb46690/solx-linux-amd64-gnu-test-build-06)
+- [Linux/Arm64](https://github.com/matter-labs/solx/releases/download/eb46690/solx-linux-arm64-gnu-test-build-06)
+- [MacOS](https://github.com/matter-labs/solx/releases/download/eb46690/solx-macosx-test-build-06)
+
+## Usage
+
+`solx` currently works by disguising itself as `solc` 0.8.29.

```

Comment:
I'd go with
We recommend using `solx` via foundry. It behaves the same way as `solc-0.8.29`, so you can download the binary and specify: